lighthouse-core/gather/gather-runner.js

```diff
@@ -100,7 +100,10 @@ class GatherRunner {
     const resetStorage = !options.flags.disableStorageReset;
     // Enable emulation based on flags
     return driver.assertNoSameOriginServiceWorkerClients(options.url)
-      .then(_ => driver.getUserAgent())
+      .then(_ => {
+        options.fetchedAt = (new Date()).toJSON();
+        return driver.getUserAgent();
```

Comment:
The other promise expects the user agent in return, throws else. Can convert to one-liner if you want to.